Protocol.md

```diff
@@ -1228,8 +1228,11 @@ A Responder that receives a REQUEST that it can not honor due to LEASE restricti
 <a name="flow-control-qos"></a>
 #### QoS and Prioritization
 
-Quality of Service and Prioritization of streams are considered application or network layer concerns and are better dealt with
+Quality of Service and Prioritization of all *non Stream ID 0* streams (all application level streams) are considered application or network layer concerns and are better dealt with
 at those layers. The metadata capabilities, including METADATA_PUSH, are tools that applications can use for effective prioritization.
+
+However, all the frames within the *Stream ID 0* **SHOULD** be considered as *highly-prioritized* frames and delivered to the connection as soon as possible. That is not necessarily a mandated part of the implementation, but a RECOMENDED one since all the related frames impact the stability and performance of the system in total.
```

Comment:
> That is not necessarily a mandated part of the implementation, but a RECOMENDED one since all the related frames impact the stability and performance of the system in total.

I think SHOULD covers this.

Reply:
yes. Metadata_Push is very important. Now we use Metadata_Push to broadcast network changes of rsocket broker cluster.

Reply:
@OlegDokuka to clarify my point, I'd delete that sentence and just rely on SHOULD or MUST.

Reply:
@yschimke, fixed! Thanks for suggestion ❤️ 🙏